Fix:

error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int
error: call to undeclared library function 'exit' with type 'void (int) __attribute__((noreturn))'; ISO C99 and later do not support implicit function declarations

https://github.com/DALnet/bahamut/pull/220
--- configure.orig	2022-08-16 10:42:18.000000000 -0500
+++ configure	2023-07-03 08:48:13.000000000 -0500
@@ -5548,10 +5548,10 @@
 #include <sys/types.h>
 #include <stdio.h>
 #include <errno.h>
-main()
+int main()
 {
   char *s = sys_errlist[0];
-	  exit(0);
+  return 0;
 }
 
 _ACEOF
